chaincode/fabcar/go/contract/shim.go

```diff
@@ -0,0 +1,21 @@
+package contract
+
+import (
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+)
+
+//go:generate counterfeiter -o mocks/transaction.go -fake-name TransactionContext . TransactionContext
+type TransactionContext interface {
+	contractapi.TransactionContextInterface
+}
```

Comment:
with this filename `shim.go`, I was confused.
It initially led me to think these types were used in main smart contract.

will removing this file, and moving all go:generate + types to file  `smartcontract_test.go` work or similar refactor? Similar to fabric repo tests. 

And make them non-exports of this package, will be better practice. so `type transactionContext `

Reply:
Correct, in the next update it was pushed into the test file itself. But then we decided to write the tests for a different chaincode so it was never pushed.

Reply:
And good call on the `unexported` interfaces